nyalib/small_helpers.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+"""
+Intended for helper functions which are needed in multiple places but to small to warrant their own file.
+"""
+
+def filterTextForSqlInjection(text: str):
```

Comment:
This is not the right way of doing this.  We should use a prepared statement and avoid the need of escaping special characters. 

Unless the py driver doesn't support it this is an improvement, but we could do better.